test/tInstall.m

```diff
@@ -0,0 +1,41 @@
+classdef tInstall < matlab.unittest.TestCase
+% Verify installation of matlab engine
+
+% Copyright 2023 Mathworks, Inc.
+
+    properties (Constant)
+        MATLABVersion = string(ver('MATLAB').Version)
```

Comment:
It might be nice to have a comment indicating what ver('MATLAB').Version and ver('MATLAB').Release are expected to look like.